lib/resources.rb

```diff
@@ -1131,6 +1131,28 @@ def get_resource(key)
                 ]
                }]}
 
+    DSTU2_DIAGNOSTIC_REPORT_BUNDLE ||= {
+        "resourceType"=>"Bundle",
+        "id"=> "b2502416-98eb-4a53-b9cc-3262c1617a71",
+        "type"=> "searchset",
+        "base"=> "https://fhir.sandboxcernerpowerchart.com/dstu2/open/d075cf8b-3261-481d-97e5-ba6c48d3b41f",
+        "total"=> 1,
+        "link"=> [{"relation"=> "self",
+                   "url"=> "https://fhir.sandboxcernerpowerchart.com/dstu2/open/d075cf8b-3261-481d-97e5-ba6c48d3b41f/DiagnosticReport?patient=1316020"}],
+        "entry"=> [{"resource"=> {"resourceType"=> "DiagnosticReport",
+                                  "id"=> "3179355",
+                                  "name"=> {"coding"=> [{"system"=> "http://snomed.info/sct",
```

Comment:
FHIR requires that we return loinc, which our service does.  The example should reflect that.